.gitattributes

```diff
@@ -38,6 +39,7 @@ dashboard/app/assets/javascripts/** !text !filter !merge !diff
 dashboard/app/assets/stylesheets/** !text !filter !merge !diff
 dashboard/config/locales/**/*en.yml !text !filter !merge !diff
 i18n/locales/source/** !text !filter !merge !diff
+pegasus/cache/i18n/en-US.json !text !filter !merge !diff
```

Comment:
Excluding the English file from `Git LFS` so PRs display changes to the English strings.